tests/markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -901,16 +901,20 @@ See [CONTRIBUTING.md](CONTRIBUTING.md).
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # Prettier
 
-[![Gitter](https://badges.gitter.im/gitterHQ/gitter.svg)](https://gitter.im/jlongster/prettier)
-[![Build Status](https://travis-ci.org/prettier/prettier.svg?branch=master)](https://travis-ci.org/prettier/prettier)
-[![Codecov](https://img.shields.io/codecov/c/github/prettier/prettier.svg)](https://codecov.io/gh/prettier/prettier)
-[![NPM version](https://img.shields.io/npm/v/prettier.svg)](https://www.npmjs.com/package/prettier)
-[![styled with prettier](https://img.shields.io/badge/styled_with-prettier-ff69b4.svg)](#badge)
+[![Gitter](https://badges.gitter.im/gitterHQ/gitter.svg)](
+https://gitter.im/jlongster/prettier) [![Build Status](
+https://travis-ci.org/prettier/prettier.svg?branch=master)](
+https://travis-ci.org/prettier/prettier) [![Codecov](
+https://img.shields.io/codecov/c/github/prettier/prettier.svg)](
+https://codecov.io/gh/prettier/prettier) [![NPM version](
+https://img.shields.io/npm/v/prettier.svg)](
+https://www.npmjs.com/package/prettier) [![styled with prettier](
+https://img.shields.io/badge/styled_with-prettier-ff69b4.svg)](#badge)
```

Comment:
These looks weird but since multi-image is not common and it can be simply achieved by `<!-- prettier-ignore -->`, should not be a problem.

Reply:
It's very common for badge lists